spec/models/inventory_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+RSpec.describe Inventory, type: :model do
+  describe 'Inventory model' do
+    let(:inventory) { FactoryBot.create(:inventory, name: 'Edward', user_id: 1) }
+  end
+
+  before { inventory_queries.save }
+
+  it 'should check for valid attributes' do
+    expect(inventory_queries).to be_valid
```

Comment:
- [ ] Can you help me understand where the `inventory_queries` variable is assigned with an inventory model object? How about we use the `inventory` object created using FactoryBot? what do you think?

Reply:
Hello @gtekle,
Thanks for the feedback I was suppose to use the `inventory` as the variable not `inventory_queries`